workout/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+class Workout(models.Model):
+    workout_name = models.CharField(max_length = 200)
+    aerobic = 1
+    strength = 2
+    balance = 3
+    flexibility = 4
+    workoutType = (
+        (aerobic, 'Aerobic exercise'),
+        (strength, 'Strength exercise'),
+        (balance, 'Balance exercise'),
+        (flexibility, 'Flexibility exercise'),
+        )
+    workout_type = models.IntegerField(choices = workoutType, blank = False, default = None)
```

Comment:
@regilkent123 @iamkatrinaanne use positive integerfield here

Reply:
also don't put spaces around `=` if its an argument and not a statement. please review reading coding with python :)